src/main/java/org/jvnet/hudson/test/PluginAutomaticTestBuilder.java

```diff
@@ -65,6 +84,7 @@ public static TestSuite build(Map<String,?> params) throws Exception {
             String packaging = StringUtils.defaultIfBlank((String)params.get("packaging"), "hpi");
             if ("hpi".equals(packaging)) {
                 inJenkins.addTest(new OtherTests("testPluginActive", params));
+                inJenkins.addTest(new OtherTests("testStaplerDispatches", params));
```

Comment:
I would rather other things did not go into `InjectedTest` (and that it would die).  The reason being is that skipping part of the tests (rather than all of them) is pretty much impossible as it uses old junit Suites rather than newer test classes, and is hard to debug.
This would be much better going forward in a `generate-tests` mojo in maven-hpi-plugin which then also makes it much easier to debug in an IDE.  but otherwise 👍